installer/pkg/config/libvirt/libvirt.go

```diff
@@ -14,12 +14,12 @@ const (
 
 // Libvirt encompasses configuration specific to libvirt.
 type Libvirt struct {
-	URI           string `json:"tectonic_libvirt_uri,omitempty" yaml:"uri"`
-	QCOWImagePath string `json:"tectonic_coreos_qcow_path,omitempty" yaml:"imagePath"`
-	Network       `json:",inline" yaml:"network"`
-	MasterIPs     []string `json:"tectonic_libvirt_master_ips,omitempty" yaml:"masterIPs"`
-	WorkerIPs     []string `json:"tectonic_libvirt_worker_ips,omitempty" yaml:"workerIPs"`
-	BootstrapIP   string   `json:"tectonic_libvirt_bootstrap_ip,omitempty" yaml:"bootstrapIP"`
+	URI         string `json:"tectonic_libvirt_uri,omitempty" yaml:"uri"`
+	QCOWImage   string `json:"tectonic_coreos_qcow,omitempty" yaml:"image"`
```

Comment:
Can we make this just `Image`?  [`libvirt_volume`'s `source`][1] doesn't have to be QCOW (does it?).  Also, this seems like a good opportunity to make the `json` serialization just `image`, in which case the Go, JSON, and YAML properties would all match :).

https://github.com/dmacvicar/terraform-provider-libvirt/blob/v0.4.4/website/docs/r/volume.html.markdown#argument-reference

Reply:
How about OSImage? I worry that image may be a little ambiguous to someone not familiar with the code or config.

Reply:
> How about OSImage? I worry that image may be a little ambiguous to someone not familiar with the code or config.

We already have some context by being a property of the `Libvirt` structure, so I don't think we need an `OS` prefix.  I'm not strongly against it though; my current ranking is `Image` > `OSImage` >> `QCOWImage` ;).

Reply:
I'll go with `Image`.